spec/support/test_factories.rb

```diff
@@ -54,6 +54,33 @@ def create_wiki_without_title
       click_on( 'Save' )
     end
   end
+
+  def click_on_upgrade_link
```

Comment:
only it would make more sense to have the helpers on their own support file (helpers.rb?), factories are something altogether different